component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java

```diff
@@ -70,9 +70,11 @@ public static void main(String[] args) {
   public void run() {
 
     try {
-      logger.info("Starting Fdo Device");
+      logger.info("Starting FDO Device");
       super.run();
-      logger.info("Starting Fdo Completed");
+    } catch (RuntimeException e) {
+      logger.error(e.getMessage());
+      logger.info("Exiting FDO Device Application");
```

Comment:
By FDO Device application, are we referring to DI ? or FDO

Reply:
FDO

Reply:
It's a common fallback path.